.github/workflows/bump.yml

```diff
@@ -72,3 +72,75 @@ jobs:
         timeout-minutes: 60
         with:
           limit-access-to-actor: true
+
+  cargo-upgrades:
+    runs-on: "lab"
```

Comment:
Does this need to run in the lab?

Reply:
the compile is really slow on the hosted runners.  It took like 6 minuts just to build cargo-deny

Reply:
Can't we find a binary for cargo-deny? Or a Docker image? We should probably avoid recompiling it everytime if we could.

But OK

Reply:
That is a good idea.  As I recall binstall was broken on the lab machines for some reason but it would be fine on the hosted runners I think

Reply:
Otherwise they publish binaries on the GH repo (https://github.com/EmbarkStudios/cargo-deny/releases/latest), although now I think about it this is maybe where `binstall` might be fetching them.